flowmachine/flowmachine/features/location/flows.py

```diff
@@ -49,6 +49,48 @@ def inflow(self):
 
         return InFlow(self)
 
+    def _build_json_agg_clause(self, loc_cols, direction, label_cols=[]):
```

Comment:
Can probably get rid of the `loc_cols` and `label_cols` args now, and infer them from `self`